src/redprl/refiner_types.fun

```diff
@@ -56,10 +56,10 @@ struct
     fun EqType _ jdg =
       let
         val _ = RedPrlLog.trace "Bool.EqType"
-        val H >> AJ.EQ_TYPE ((a, b), l, k) = jdg
+        val H >> ajdg = jdg
+        val ((a, b), _, k) = viewAsEqType ajdg
```

Comment:
@jonsterling So this is the new programming style to handle both `EQ` and `EQ_TYPE` at once.

Reply:
cool! That sounds reasonable.